src/auth/auth.service.ts

```diff
@@ -160,7 +160,7 @@ export class AuthService {
   // Parameters:
   //    token: both the pure jwt token and the one with "Bearer " or "bearer " are supported.
   verify(token: string): Authorization {
-    if (token == undefined || token == undefined || token == '')
+    if (token == undefined || token == '')
```

Comment:
if you notate `token: string`, then token should never be undefined
if there is a chance token is undefined, then the token should be `string | undefined` type (or more concisely, `token?: string`)

Reply:
https://github.com/SageSeekerSociety/cheese-backend/commit/b2f1f1f3851f81eb537879486fb97e4c81e76fec fixed